gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/PlanEntity.java

```diff
@@ -36,24 +37,30 @@ public class PlanEntity {
      */
     private String crossId;
 
+    @NotNull
```

Comment:
I defined this annotation based on the one set in the CreatePlan and UpdatePlan entity